### PR TITLE
Fixed watchman install link

### PR DESCRIPTION
**Pull request by hellojustxn**
Fixed "Page not found" link in get-started doc. Hyperlink now points to the correct install page for Watchman.

Modified in v36.0.0

# Why

FB changed the url to the watchman installation page.

# How

Removed .html from original URL

https://facebook.github.io/watchman/docs/install

# Test Plan

Open https://facebook.github.io/watchman/docs/install

### CODE REVIEW

File: docs/pages/versions/v36.0.0/get-started/installation.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Verify that the installation was successful by running `expo whoami`. You're not
 
 ### Optional: Installing Watchman
 
-Some macOS users encounter issues if they do not have Watchman installed on their machine, so if you are using a Mac we recommend that you install it. [Download and install Watchman](https://facebook.github.io/watchman/docs/install.html).
+Some macOS users encounter issues if they do not have Watchman installed on their machine, so if you are using a Mac we recommend that you install it. [Download and install Watchman](https://facebook.github.io/watchman/docs/install).
```
Comment (cruzach) [inline docs/pages/versions/v36.0.0/get-started/installation.md:30]:
```suggestion
Some macOS users encounter issues if they do not have Watchman installed on their machine, so if you are using a Mac we recommend that you install it. [Download and install Watchman](https://facebook.github.io/watchman/docs/install#buildinstall).
```

**Review by cruzach — CHANGES_REQUESTED** [review]
thanks! can you make this change to the `unversioned` and `v37.0.0` docs?

File: docs/pages/versions/v36.0.0/get-started/installation.md (hunk 1 above)
Comment (justingwl) [reply docs/pages/versions/v36.0.0/get-started/installation.md:30]: Hi @cruzach - I just made the changes to the `unversioned` and `v37.0.0` docs in #7827  and #7828

**Comment by cruzach** [comment]
@justingwl can you make the commits from 7827 and 7828 in this PR instead? no need to put it in 3 separate PRs

**Review by hellojustxn — COMMENTED** [review]
@cruzach Ok, I made the commits for updating v36.0.0, v37.0.0, and unversioned here

**Comment by cruzach** [comment]
thanks 🎉 you rock!

**Pull request merged**